app/assets/stylesheets/general_shopping.css

```diff
@@ -0,0 +1,15 @@
+/* .shopping_table {
+  margin: 0 auto;
+}
+
+.shopping_table,
+.shopping_heading,
+.shopping_detail {
+  border: 1px solid #929292;
+  border-collapse: collapse;
+  padding: 3px;
+}
+
+.shopping_even {
+  background: rgba(0, 0, 0, 0.2);
+} */
```

Comment:
- [ ] Kindly Remove all commented code to make your code clear and clean.

Reply:
Thanks, I have remove the unnecessary code.